woocommerce-services.php

```diff
@@ -1801,6 +1821,20 @@ public function are_woo_shipping_and_woo_tax_active() {
 		public function display_woo_shipping_and_woo_tax_are_active_notice() {
 			echo '<div class="error"><p><strong>' . esc_html__( 'Woo Shipping and Woo Tax plugins are already active. Please deactivate WooCommerce Shipping & Tax.', 'woocommerce-services' ) . '</strong></p></div>';
 		}
+
+		public function display_wcst_to_wcshipping_migration_notice() {
+			echo wp_kses_post(
```

Comment:
`woocommerce-services/assets/stylesheets/style.scss` isn't loaded on the orders page. We could add some hooks to load a separate stylesheet for this notice, but I chose to use inline styles instead because this notice is a standalone component and its style is not reusable by others yet. If needed, we can always refactor afterwards.

Reply:
👋 Hey @Ferdev , based on the design feedback p1718744494706379-slack-C06FP5XT1S5 and p1718745005310189-slack-C06FP5XT1S5, I think we will have to change the "circle-x" to "x" (still confirming). 

If we need to replace it, I think I would pull all these css out into its own `.scss` file. This will make it much easier to overwrite the `button.is-dismissible` class. To create a css file that loads in the order listing page, I think the work involves:
- Add a new scss file like `assets/stylesheets/wcst-migration-wcshipping.scss`. 
- Figure out webpack: Webpack didn't build that `.scss` properly when I tried it, even though the [config](https://github.com/Automattic/woocommerce-services/blob/9e93c9fda94b1650a4a3d1fba8e1560a1d9fb8fa/webpack.config.js#L165) should build it. 
- I ended up adding an entry point in `.js`. Then do `import '../assets/stylesheets/wcst-migration-wcshipping.scss';`. This seems to work. We will then have to `wp_enqueue_script` and `wp_enqueue_style` these extra scripts for the css. I'm not sure if there is a better way.
- Consider if we should build our own dismiss button if it's hard to overwrite the default styles.

Reply:
hmm, webpack not building that css file is weird, apparently the `sass-loader` is looking for stuff in that path. Could be the problem the file is missing the css extension? I've seen sometimes sass preprocessors not liking filenames with just the sass extension, ie, `my-css-file.scss` instead of just `my-css-file.css.scss`.

About creating that follow up: yeah, I think it's the best approach. This way we can unlock this PR and rebase mine with it :+1: